misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -3007,12 +3008,14 @@ public ReceiveActionNode transform(ReceiveActionNode receiveActionNode) {
 
     @Override
     public ReceiveFieldsNode transform(ReceiveFieldsNode receiveFieldsNode) {
-        Token openBrace = formatToken(receiveFieldsNode.openBrace(), 0, 1);
+        int trailingNL = shouldExpand(receiveFieldsNode) ? 1 : 0;
+        Token openBrace = formatToken(receiveFieldsNode.openBrace(), 0, trailingNL);
```

Comment:
consider the below sample. The indentation for the open brace is not set correctly. 

Before formatting:
```bal
function foo() {
    do {
        _ = <-
{
a: w1,b: w2
};
    }
}
```

After formatting:
```bal
function foo() {
    do {
        _ = <-
{
            a: w1,
            b: w2
        };
    }
}
```

Reply:
@poorna2152 can you point the created issue for this here?

Reply:
Ack.. The generic formatter issue [here](https://github.com/ballerina-platform/ballerina-lang/issues/41895)